.github/workflows/tests.yml

```diff
@@ -180,3 +180,41 @@ jobs:
           --splits=6 --group=${{ matrix.group }} --durations-path=../../.github/.test_durations
           tests ../datachain/tests
         working-directory: backend/datachain_server
+
+
+  examples:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest-16-cores, macos-latest, windows-latest-8-cores]
+        pyv: ['3.9', '3.12']
+        group: ['get_started', 'llm_and_nlp or computer_vision', 'multimodal']
+    steps:
+
+      - uses: actions/checkout@v4
+
+      - name: Set up Python ${{ matrix.pyv }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.pyv }}
+          cache: 'pip'
+
+      - name: Install libmagic for unstructured package
+        if: ${{ runner.os == 'macOS' &&  matrix.group == 'llm_and_nlp or computer_vision' }}
+        run: brew install libmagic
```

Comment:
just to double check - do we still need libmagic?

Reply:
yes, this is a pre-requirement of `unstructured`

Reply:
we can simplify it with `from unstructured.partition.pdf import partition_pdf as partition` I think (since we deal only with PDFs)